integration/tests/cook/test_cli.py

```diff
@@ -1729,7 +1729,8 @@ def test_entity_refs_duplicate_uuid(self):
         self.assertEqual(1, len(groups))
 
     def test_entity_refs_case_insensitive(self):
-        config = {'clusters': [{'name': 'Foo', 'url': self.cook_url}]}
+        config = {'clusters': [{'name': 'Foo', 'url': self.cook_url}],
+                  'defaults': {'submit': {'cluster': '', 'pool-name': '', 'url': ''}}}
```

Comment:
There were a few tests where I could not avoid clearing out the default submit cluster, so I updated them.